.buildbot.sh

```diff
@@ -56,13 +56,13 @@ cmake -DCMAKE_INSTALL_PREFIX=`pwd`/../inst \
     ../llvm
 cmake --build .
 cmake --install .
-export PATH=`pwd`/../inst/bin:${PATH}
+export YKB_YKLLVM_INSTALL_DIR=`pwd`/../inst/bin
```

Comment:
Why not let the build use the submodule? Then all the duplicated cmake args above can be deleted.

Reply:
> Why not let the build use the submodule?

I don't understand what that means.

Reply:
Hopefully 21b2f5e clarifies the motivation?

Reply:
Yes, thanks!